depthai_ros_driver/CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ add_library (${PROJECT_NAME}_pipelines
     src/pipelines/preview_pipeline.cpp
     src/pipelines/stereo_pipeline.cpp
     src/pipelines/mobilenet_ssd_pipeline.cpp
+    src/pipelines/stereo_with_detection_pipeline.cpp
```

Comment:
Thoughts on stereo_with_mobilenet_ssd_pipeline?

Reply:
renamed in latest commit